install error

after running 
make get_glide
make install_dependencies
i run into the following error on make build
```
gofmt -s -w .
vendor/golang.org/x/net/ipv4/batch.go:59:14: expected type, found '='
vendor/golang.org/x/net/ipv4/batch.go:78:2: expected declaration, found 'if'
vendor/golang.org/x/net/ipv6/batch.go:56:14: expected type, found '='
vendor/golang.org/x/net/ipv6/batch.go:69:2: expected declaration, found 'if'
Makefile:8: recipe for target 'format' failed
make: *** [format] Error 2
```

# make build
gofmt -s -w .
vendor/golang.org/x/net/ipv4/batch.go:59:14: expected type, found '='
vendor/golang.org/x/net/ipv4/batch.go:78:2: expected declaration, found 'if'
vendor/golang.org/x/net/ipv6/batch.go:56:14: expected type, found '='
vendor/golang.org/x/net/ipv6/batch.go:69:2: expected declaration, found 'if'
Makefile:8: recipe for target 'format' failed
make: *** [format] Error 2

Followed the instruction Given By @Minitram 

# make build
gofmt -s -w .
go-bindata -o cmd/xray/ui.go -pkg main ui
make: go-bindata: Command not found
Makefile:34: recipe for target 'static' failed
make: *** [static] Error 127

# go get -u github.com/jteeuwen/go-bindata/...

# make build
gofmt -s -w .
go-bindata -o cmd/xray/ui.go -pkg main ui
make: go-bindata: Command not found
Makefile:34: recipe for target 'static' failed
make: *** [static] Error 127

I got same error while try to build docker 

docker build -t xraydocker .
....
....
....
gofmt -s -w .
vendor/golang.org/x/net/ipv4/batch.go:59:14: expected type, found '='
vendor/golang.org/x/net/ipv4/batch.go:78:2: expected declaration, found 'if'
vendor/golang.org/x/net/ipv6/batch.go:56:14: expected type, found '='
vendor/golang.org/x/net/ipv6/batch.go:69:2: expected declaration, found 'if'
make: *** [Makefile:8: format] Error 2

so why this issue is marked as closed?

Another different error even after the Minitram fix.
```
➜  xray git:(master) make build
gofmt -s -w .
go-bindata -o cmd/xray/ui.go -pkg main ui
# command-line-arguments
cmd/xray/bfs.go:57: undefined: AssetDir
cmd/xray/bfs.go:57: undefined: AssetInfo
Makefile:39: recipe for target 'build' failed
make: *** [build] Error 2
➜  xray git:(master) go get -u github.com/jteeuwen/go-bindata/...
➜  xray git:(master)  cd $GOPATH/src/github.com/jteeuwen/go-bindata/go-bindata
➜  go-bindata git:(master) ✗ go build
➜  go-bindata git:(master) ✗ cd ~/go/src/github.com/evilsocket/xray
➜  xray git:(master) make build
gofmt -s -w .
go-bindata -o cmd/xray/ui.go -pkg main ui
# command-line-arguments
cmd/xray/bfs.go:57: undefined: AssetDir
cmd/xray/bfs.go:57: undefined: AssetInfo
Makefile:39: recipe for target 'build' failed
make: *** [build] Error 2
➜  xray git:(master) go version
go version go1.8.3 linux/amd64

```
Probably related with this issue.

--- a/Makefile
+++ b/Makefile
@@ -2,10 +2,11 @@ NAME=xray
 SOURCE=cmd/$(NAME)/*.go
 GOBUILD=go build
 DEPEND=github.com/Masterminds/glide
+GOFILES=$(shell find . -type f -name '*.go' -not -path "./vendor/*")
 
 .PHONY: format
 format:
-	gofmt -s -w .
+	@gofmt -s -w $(GOFILES)
 
 # Command to get glide, you need to run it only once
 .PHONY: get_glide

